zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -225,6 +229,49 @@ private void removeExporter(final String exporterId, final ExporterContainer con
     }
   }
 
+  /**
+   * Enables an exporter with the given id and descriptor. The exporter will start exporting records
+   * after this operation completes.
+   *
+   * <p>It is expected that the exporter to initialize from the metadata is of same type as the
```

Comment:
:thought_balloon: We should ensure that the caller does that, I guess in the next PR